Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Create builder image
+FROM node:16-alpine AS builder
```

Comment:
Can you add the `docker.io/library/` prefix, so that it's explicit from where it's coming from?